test/modules/geo-layers/tile-layer/tile-2d-header.spec.js

```diff
@@ -46,3 +46,21 @@ test('Tile2DHeader#Cancel request if not selected', async t => {
   t.ok(tile2.isCancelled, 'Second request was cancelled');
   t.end();
 });
+
+test('Tile2DHeader#Abort quickly', async t => {
+  const requestScheduler = new RequestScheduler({throttleRequests: true, maxRequests: 1});
+
+  const getTileData = () => null;
+  const onTileLoad = () => null;
+  const onTileError = () => null;
+
+  const tile = new Tile2DHeader({onTileLoad, onTileError});
+
```

Comment:
Add `tile.isSelected = true;` so that a request is actually sceduled

Reply:
https://github.com/visgl/deck.gl/pull/5044/files#diff-5dd332fb839a9666cc7d7489bfa32eda9f101a0f17b44034ea90a5d132d61b74R58